src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -139,6 +139,51 @@ Copyright (c) .NET Foundation. All rights reserved.
           TaskParameter="TargetOutputs"
           ItemName="_ProjectReferences" />
     </MSBuild>
+
+    <MSBuild
+      Condition="'$(PackBuildOutput)' == 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="BuiltProjectOutputGroup"
+      Properties="TargetFramework=%(_TargetFrameworks.Identity)
+              %(_MSBuildProjectReferenceExistent.SetConfiguration);
+              %(_MSBuildProjectReferenceExistent.SetPlatform);
+              BuildProjectReferences=false;"
+      RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+
+      <Output
+          TaskParameter="TargetOutputs"
+          ItemName="_TargetPathsToAssemblies" />
+    </MSBuild>
+
+    <MSBuild
+      Condition="'$(IncludeSymbols)' == 'true' OR '$(IncludeSource)' == 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="DebugSymbolsProjectOutputGroup"
+      Properties="TargetFramework=%(_TargetFrameworks.Identity)
+              %(_MSBuildProjectReferenceExistent.SetConfiguration);
+              %(_MSBuildProjectReferenceExistent.SetPlatform);
+              BuildProjectReferences=false;"
+      RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+
+      <Output
+          TaskParameter="TargetOutputs"
+          ItemName="_TargetPathsToPDB" />
```

Comment:
this is an internal variable, so wouldn't have mattered, but i'll change it anyway.